Source/Bogus/Extensions/Chile/ExtensionsForChile.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using Bogus.DataSets;
+
+namespace Bogus.Extensions.Chile
+{
+    /// <summary>
+    /// API extensions specific for a geographical location.
+    /// </summary>
+    public static class ExtensionsForChile
+    {
+      /// <summary>
+      /// Generates a valid Chilean RUT for a Person (Rol Unico Tributario)
+      /// </summary>
+      /// <param name="dotFormat">Use the thousands separator for the Chilean RUT (11.111.111-1)</param>
+      /// <returns>A string representation for a valid Chilean RUT (Rol Unico Tributario)</returns>
+      public static string Rut(this Person person, bool dotFormat = false)
+      {
+         return GenerateChileanId(dotFormat);
+      }
+
+      /// <summary>
+      /// Generates a valid Chilean RUT for a Company (Rol Unico Tributario)
+      /// </summary>
+      /// <param name="dotFormat">Use the thousands separator for the Chilean RUT (11.111.111-1)</param>
+      /// <returns>A string representation for a valid Chilean RUT (Rol Unico Tributario)</returns>
+
+      public static string Rut(this Company company, bool dotFormat = true)
+      {
+         return GenerateChileanId(dotFormat);
+      }
+
+      /// <summary>
+      /// A general Chilean ID generator
+      /// </summary>
+      /// <param name="dotFormat">Use the thousands separator for the Chilean RUT (11.111.111-1)</param>
+      /// <returns></returns>
+      private static string GenerateChileanId(bool dotFormat = true)
+      {
+         Random rnd = new();
+         int num = rnd.Next(1000000, 99999999);
```

Comment:
We shouldn't call `new Random()` in Bogus because it's not deterministic. We'll need to source the `rnd` from `Person.Random` to get the randomizer, and then ask for the `rnd.Next()`.